deploy/before_restart

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+oldrev, newrev = ARGV
+
+def run(cmd)
+  exit($?.exitstatus) unless system "umask 002 && #{cmd}"
+end
+
+RAILS_ENV   = ENV['RAILS_ENV'] || 'production'
```

Comment:
Hey. The file is autogenerated by https://github.com/mislav/git-deploy - I'd prefer to keep it the same if possible. But I'm happy to fall back to an invalid environment instead if you want :)